frontend/src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,42 @@
+import { TouchEvent, useRef, useState } from 'react'
+import {
+  Box,
+  Tooltip as ChakraTooltip,
+  TooltipProps,
+  useMultiStyleConfig,
+} from '@chakra-ui/react'
+
+import { TOOLTIP_THEME_KEY } from '~/theme/components/Tooltip'
+
+export const Tooltip = ({ children, ...props }: TooltipProps): JSX.Element => {
+  // component is controlled in mobile mode
```

Comment:
Do we need special handling in mobile? I tried Chakra's own tooltip on mobile and it seems to work fine

Reply:
oh noi tried tapping with chrome's dev tool device emulator and it didn't work :( didn't think of trying on my phone. Will remove!